docs/src/main/paradox/migrations.md

```diff
@@ -46,6 +49,7 @@ which is likely to time out. You can also use this method to stagger your migrat
 You do not need to worry if a small number of events are missed by `migrateToTagViews` as they will be
 fixed during your `PersistentActor` recovery. However do not rely on this for full migration as only active `PersistentActor`s
 will be recovered and it will mean the start up time for your `PersistentActor`s will be very long.
+Moreover, if your application consumes events through `eventsByTag` query, the missed events will only show up on the query side once you instantiate the respective `PersistentActor`. 
```

Comment:
Lagom users will be impacted by that, so better to mention in migration guide that query side may be impacted. 

In one of my tests, I run the migration, added some new events and then restart the app using 0.99. The missed events were not showing up on the query side, of course. Then, I re-run the events migration while the app was running and the query side started to 'see' the missed events. 
However, I don't know how it will behave if we re-instantiate a `PersistentActor` while the events migration is running on another process. @chbatey, will it be safe to run it?

Reply:
It isn't safe to run the migration while 0.99 is running. As then there will be two writers for the same persistenceId into the new tag views table

Reply:
Sure, I suspected that. But was in the hope for some kind of leasing. Anyway, it's noted. Thanks!